docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -250,10 +270,16 @@ PUT /_data_stream/logs_alt
 [[get-info-about-a-data-stream]]
 === Get information about a data stream
 
-You can use the <<indices-get-data-stream,get data stream API>> to get
-information about one or more data streams, including:
+To view information about a data stream in Kibana, open the menu and go to
+*Stack Management > Index Management*. In the *Data Streams* tab, click a data
+stream's name to view information about the stream.
+
+[role="screenshot"]
+image::images/data-streams/data-streams-list.png[]
+
+You can also use the <<indices-get-data-stream,get data stream API>> to retrieve
+the following information about one or more data streams:
```

Comment:
```suggestion
You can also use the <<indices-get-data-stream,get data stream API>> to verify that a recently created data stream
exists, and to retrieve the following information about one or more data streams:
```

Reply:
This edit seeks to combine the thought on [line 291](https://github.com/elastic/elasticsearch/pull/60118/files#diff-1cb4ce901c68835c9f095b60faf0661eR291) and remove the "handy" reference that might be colloquial. Outside the realm of your PR though.